src/app/(Authentication)/Login/Email/index.tsx

```diff
@@ -32,7 +34,7 @@ export default function LoginScreen() {
   return (
     <View style={styles.container}>
       <TouchableOpacity style={styles.backButton} onPress={() => router.back()}>
```

Comment:
Can we add a `contentContainer` View that defines the width of the content on this screen. That way all components have a consistent width.